apps/backoffice-v2/src/pages/Entity/hooks/useEntity/childContexts.ts

```diff
@@ -0,0 +1,228 @@
+export const childContexts = [
```

Comment:
The mock parent workflow with children which was used during development, will delete it in the part 2/2 PR.